cypress/features/regression/accessibility/accessibilityDesignSystem.feature

```diff
@@ -205,4 +205,15 @@ Feature: Accessibility tests - Design System folder
   @accessibility
   Scenario: Component Toast
     When I open "Design System Toast Test" component page "visual" in no iframe
-    Then "Toast" component has no accessibility violations
\ No newline at end of file
+    Then "Toast" component has no accessibility violations
+
+  @accessibility
+  Scenario Outline: Component Card <story>
+    When I open "Design System Card" component page "<story>" in no iframe
+    Then "Card <story>" component has no accessibility violations
+    Examples:
+      | story                         |
+      | default_story                 |
+      | interactive                   |
+      | different_card_footer_padding |
+      | more_exampels_of_card_footer  |
```

Comment:
examples is spelt wrong. This should be changed in the mdx as well 
```suggestion
      | more_examples_of_card_footer  |
```
this should be changed in the mdx as well so that the accessibility tests run okay. I think they will initially fail because the phone buttons have don't have "discernible text". See the story and accessibility tab for more info